client/src/components/history/history-chat.tsx

```diff
@@ -183,6 +184,7 @@ export function HistoryChat(args: ScrollingQuestionsParams): JSX.Element {
                   setQuestionVisibilityPref(m.questionId, show);
                 }}
                 visibility={isQuestionsAnswersVisible(m.questionId)}
+                config={config}
```

Comment:
there should be no reason ever to pass config from component to component. If these items need something from config, they should `useSelector` to get it themselves

Reply:
fixed